File: korge-spine/src/jvmMain/java/com/badlogic/gdx/graphics/Mesh.java
package com.badlogic.gdx.graphics;

import com.badlogic.gdx.graphics.glutils.ShaderProgram;

public class Mesh {
    public Mesh(VertexDataType vertexDataType, boolean b, int maxVertices, int i, VertexAttribute a_position, VertexAttribute a_light, VertexAttribute a_dark, VertexAttribute a_texCoord0) {
    }

    public void setVertices(float[] vertices, int i, int vertexIndex) {

    }

    public void setIndices(short[] triangles, int i, int triangleIndex) {

    }

    public void render(ShaderProgram shader, int glTriangles, int i, int triangleIndex) {

    }

    public void dispose() {
    }

    public enum VertexDataType {
        VertexArray, VertexBufferObject, VertexBufferObjectSubData, VertexBufferObjectWithVAO
    }
}

File: korge-spine/src/jvmMain/java/com/badlogic/gdx/graphics/Mesh.kt
package com.badlogic.gdx.graphics

import com.badlogic.gdx.graphics.Mesh.VertexDataType
import com.badlogic.gdx.graphics.VertexAttribute
import com.badlogic.gdx.graphics.glutils.ShaderProgram

class Mesh(vertexDataType: VertexDataType?, b: Boolean, maxVertices: Int, i: Int, a_position: VertexAttribute?, a_light: VertexAttribute?, a_dark: VertexAttribute?, a_texCoord0: VertexAttribute?) {
    fun setVertices(vertices: FloatArray?, i: Int, vertexIndex: Int) {}
    fun setIndices(triangles: ShortArray?, i: Int, triangleIndex: Int) {}
    fun render(shader: ShaderProgram?, glTriangles: Int, i: Int, triangleIndex: Int) {}
    fun dispose() {}
    enum class VertexDataType {
        VertexArray, VertexBufferObject, VertexBufferObjectSubData, VertexBufferObjectWithVAO
    }
}
